scripts/campaigns/monitor_coevolution_pilot.sh

```
@@ -0,0 +1,191 @@
+    exit 1
+fi
+
+ROOT="$1"
+if [ ! -d "$ROOT" ]; then
+    echo "Error: $ROOT is not a directory." >&2
+    exit 1
+fi
+
+# Discover every session dir (one that contains `prey/lineage.csv`).
+# `find` is the simplest way to handle all four layouts uniformly.
+mapfile -t SESSIONS < <(find "$ROOT" -type f -name "lineage.csv" -path "*/prey/lineage.csv" -exec dirname {} \; | xargs -I{} dirname {} | sort -u)
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)